File: src/Transaction.php
<?php
/**
 * Transaction class | src/Transaction.php.
 *
 * @author Stormix <madadj4@gmail.com>
 * @license MIT
 *
 * @version 0.1
 *
 * @copyright Copyright (c) 2018, Stormix.co
 */

namespace infiniteCoin;

/**
 * Transaction
 *
 */
class Transaction
{
    public function __construct($fromAddress, $toAddress, $amount){
        $this->fromAddress = $fromAddress;
        $this->toAddress = $toAddress;
        $this->amount = $amount;
    }
}

File: infiniteCoin/Transaction.py
"""
/**
* @author Stormix <madadj4@gmail.com>
* @license MIT
*
* @version 0.1
*
* @copyright Copyright (c) 2018, Stormix.co
*/
"""
from collections import OrderedDict


class Transaction():
    def __init__(self, fromAddress, toAddress, amount):
        self.fromAddress = fromAddress
        self.toAddress = toAddress
        self.amount = amount

    def transactionDict(self):
        return OrderedDict({"fromAddress": self.fromAddress, "toAddress": self.toAddress, "amount": self.amount})
